packages/gboost-ui/src/FileUpload/README.md

```diff
@@ -0,0 +1,181 @@
+# File Upload
+
+## Permissons
+
+### Content Security Policy (CSP)
+
+* "*.s3.[REGION].amazonaws.com/" must be a source for connect-src
+
+### Bucket CORS Policy
+
+* Bucket's CORS policy must include exposedHeaders: ["ETag"] and allowedMethods: ["PUT"]
+
+``` 
+const exampleBucket = new Bucket(this, "example-bucket", {
+    cors:[{
+    allowedHeaders: ["*"],
+    allowedMethods: ["PUT"],
+    allowedOrigins: ["*"],
+    exposedHeaders: ["ETag"],
+    }],
+});
+```
```

Comment:
Do you think you could check for these requirements at CDK synth time within the function construct and print out warnings for the developer if they don't have their bucket configured correctly? I would only print out warnings if exposedHeaders doesn't include "ETag", allowedMethods doesn't include "PUT", if allowedOrigins is an empty array or undefined, and allowedHeaders is an empty array or undefined. Make sure to not print false positive warnings.